testing/scripts/playwright_test.py

```
@@ -2247,6 +2250,520 @@ async def test_stylelint_clean(page: Page) -> None:  # noqa: ARG001
+        errors.clear(); warnings.clear()
+
+        # 6. Hover every help-bubble icon (tooltip JS runs)
+        await navigate(page, APP_URL)
+        await page.wait_for_timeout(200)
+        icon_count = await page.locator(".help-bubble-icon").count()
+        for i in range(icon_count):
+            icon = page.locator(".help-bubble-icon").nth(i)
+            try:
+                await icon.hover(timeout=1000)
+            except Exception:
+                pass  # off-screen icons in hidden panels are fine to skip
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)